aws-flow/lib/aws/decider/data_converter.rb

```diff
@@ -38,16 +38,7 @@ def dump(object)
       #
       def load(source)
         return nil if source.nil?
-        output = YAML.load source
-        if output.is_a? Exception
-          documents = YAML.load_stream(source)
-          if YAML.const_defined?(:ENGINE) && YAML::ENGINE.yamler == 'syck'
-            documents = documents.documents
-          end
-          backtrace = documents.find {|x| ! x.is_a? Exception}
-          output.set_backtrace(backtrace.to_a)
-        end
```

Comment:
This part is setting the backtrace back to the exception object, seems like we don't need it.
```
2.7.3 :011 > exemplar = Exception.new('exception')
 => #<Exception: exception> 
2.7.3 :012 > exemplar.set_backtrace(caller)
 => ["/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/workspace.rb:114:in `eval'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/workspace.rb:114:in `evaluate'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/context.rb:459:in `evaluate'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb.rb:541:in `block (2 levels) in eval_input'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb.rb:704:in `signal_status'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb.rb:538:in `block in eval_input'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/ruby-lex.rb:166:in `block (2 levels) in each_top_level_statement'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/ruby-lex.rb:151:in `loop'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/ruby-lex.rb:151:in `block in each_top_level_statement'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/ruby-lex.rb:150:in `catch'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/ruby-lex.rb:150:in `each_top_level_statement'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb.rb:537:in `eval_input'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb.rb:472:in `block in run'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb.rb:471:in `catch'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb.rb:471:in `run'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb.rb:400:in `start'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/bin/irb:23:in `load'", "/Users/toshio/.rvm/rubies/ruby-2.7.3/bin/irb:23:in `<main>'"] 
2.7.3 :013 > dumped = YAML.dump(exemplar)
 => "--- !ruby/exception:Exception\nmessage: exception\nbacktrace:\n- \"/Users/toshio/.rvm/rubies/ruby-2.7.3/lib/ruby/2.7.0/irb/workspace.rb:114... 
2.7.3 :014 > e = YAML.load(dumped)
 => #<Exception: exception> 
2.7.3 :016 > e.message
 => "exception" 
2.7.3 :017 > e.backtrace == exemplar.backtrace
 => true 
```